meson.build

```diff
@@ -171,8 +171,9 @@ src_libratbag = [
 	'src/driver-hidpp10.c',
 	'src/driver-logitech-g300.c',
 	'src/driver-logitech-g600.c',
-	'src/driver-roccat.c',
+ 	'src/driver-roccat.c',
```

Comment:
weird diff

Reply:
I can't seem to fix this. I reset the file to the one in master. and then added the 1 line for the new driver. and it still has this diff.

Reply:
You have a space before the tag.

![libratbag-pr-witespace](https://user-images.githubusercontent.com/11718923/84573295-8b7b4500-ad97-11ea-803e-7dcb3e92f48c.png)